Signal/src/call/CallAudioService.swift

```diff
@@ -418,7 +418,11 @@ struct AudioSource: Hashable {
 
         guard let availableInputs = session.availableInputs else {
             // I'm not sure when this would happen.
-            owsFail("No available inputs or inputs not ready")
+            if #available(iOS 9.0, *) {
```

Comment:
Isn't this test backwards?  I feel like the condition will be true on iOS >= 9.

Reply:
I still am not sure why this would happen, so I'd like to be alerted by an assert when it occurs.

Currently it's only *known* to occur on iOS8, where I'd like to ignore the assertionFailure, but I'd still like to know when this assert fails on iOS9+

I'll try to make the comment clearer.